### DISCUSSION TITLE
Create organization profiles using internal repos

### DISCUSSION THREAD

**Discussion by julian-wendt** [community/community#8461] [Other Feature Feedback, Questions, & Ideas]
I love the possibility that we now can create readmes as organization profiles! :fireworks: It is the perfect spot to inform organization members about necessary topics like structure, usage and other topics. Unfortunately, in our case, the content is not relevant to the public :worried:

Is there also a plan to enable this for internal repos? :pray:

**Comment by wesselkranenborg** [comment]
This would indeed be beneficial.

**Comment by jason-dour** [comment]
Absolutely need this capability.

**Comment by bisdevres** [comment]
I need this functionality too
